src/main.ts

```diff
@@ -978,6 +1001,24 @@ function generateEncode(ctx: Context, fullName: string, messageDesc: DescriptorP
   return joinCode(chunks, { on: '\n' });
 }
 
+/** Creates a function to encode a message by loop overing the tags then delimits the encoding with a length prefix */
+function generateEncodeDelimited(ctx: Context, fullName: string, messageDesc: DescriptorProto): Code {
+  const { options, utils, typeMap } = ctx;
+  const chunks: Code[] = [];
+
+  // create the basic function declaration
+  chunks.push(code`
+    encodeDelimited(
+      ${messageDesc.field.length > 0 ? 'message' : '_'}: ${fullName},
```

Comment:
Really minor, but I think this `message || _` variable name was for `encode` whose `for` loop would actually never use `message` if there are no fields. For `encodeDelimited`, the function body always uses `message`, so this can be just `message: ${fullName}`

Reply:
Removed in b0f5e38, thanks!